### PR TITLE
[ADD] website_sale_require_login, website_sale_suggest_create_account for 9.0

### PR DESCRIPTION
**Pull request by iledarn**
The modules "website_sale_require_login" and "website_sale_suggest_create_account" are all right for 9.0 without any modifications

### CODE REVIEW

File: website_sale_require_login/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Require login to checkout",
+    "summary": "Force users to login for buying",
+    "version": "8.0.1.0.0",
```
Comment (yelizariev) [inline website_sale_require_login/__openerp__.py]: update version to 9.0.1.0.0

**Comment by rschnapka** [comment]
Just for info. Installed and tested, works as it should. Thanks. Will try to provide German translations soon.

**Comment by pedrobaeza** [comment]
You should respect commit history following this guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0.

And please only one module migrated per PR.

**Comment by lasley** [comment]
Superseded by #124 & #125 - this can be closed.

**Pull request closed without merging**